### ISSUE TITLE
Нет транзакций

### ISSUE THREAD

**Issue by birhoff** [bem/bem-mvc#109]
Хотелось бы что-то вроде silent или лучше транзакции.

Ситуация следующая у меня есть кнопка которая бегает по модели и делает кучу изменений. Есть блок где то наверху который слушает изменения модели и отправляет аякс запросы. И таким образом без заглушек с таймаутами я получаю кучу запросов.

В идеале это решается транзакциями, но можно и silent режимом

**Comment by dima117** [comment]
а `update` не подойдет?
```
model.update({
    field1: value1,
    field2: value2
});
```
И слушать событие update, а не change.

**Comment by dosyara** [comment]
@dima117 нет, потому, что во время `update` триггеряться `change` на полях
